docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -2506,7 +2507,7 @@ from **200 OK** to **101 UPGRADED** and resends the same headers.
 
 ## 3.3 CORS Requests
 
-To set cross origin requests to the remote api please give values to 
+To set cross origin requests to the remote api please give values to
```

Comment:
I don't think the markdown engine (hugo?) does anything with this whitespace, I tried restoring it and it's not forcing a newline here.